src/components/SelectionList/BaseSelectionList.js

```diff
@@ -169,6 +169,7 @@ function BaseSelectionList({
     };
 
     const selectRow = (item, index) => {
+        textInputRef.current.focus();
```

Comment:
Can we guarantee that `textInputRef.current` will always be available in this component?